lib/improver/blending/blend_across_adjacent_points.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
I guess this filename is to accommodate different functions for the adjacent weighting beyond just triangular?
e.g. the exponential decay suggested in Nigel's document